### PR TITLE
Gitlab touchup

### PR DESCRIPTION
**Pull request by SkySkimmer**

### CODE REVIEW

**Comment by Zimmi48** [comment]
Can you add cross-references between the GitHub issue and the test case removal (so that the test is activated again if the problem gets solved).

**Comment by SkySkimmer** [comment]
See #5992.

**Comment by Zimmi48** [comment]
I meant write the issue number in the comment and write that this test is being disabled on GitLab in the issue thread.

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -114,6 +114,7 @@ before_script:
   script:
     - cd test-suite
     - make clean
+    - rm -rf coq-makefile/findlib-package # randomly fails
```
Comment (williammpratt) [inline .gitlab-ci.yml]: What kind of ridiculous comment is "randomly fails"? 

All it communicates to me is that you are ignorant about what your system is doing.

Comment (ejgallego) [reply .gitlab-ci.yml]: You are clearly not very good at communication, so it seems.

**Comment by maximedenes** [comment]
@Zimmi48 Do you request changes? Or can I merge?

**Comment by SkySkimmer** [comment]
Please don't merge, the test is showing a real bug (#6475) so it shouldn't be disabled I think (I got confused with the previous failures on the same test).

**Comment by SkySkimmer** [comment]
I popped the test removal, feel free to merge.

**Comment by Zimmi48** [comment]
@SkySkimmer Do you wish this to be backported?

**Comment by SkySkimmer** [comment]
>@SkySkimmer Do you wish this to be backported?

Yes please.

**Pull request merged**